app/views/admin/applications/show.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= render partial: '/layouts/application_header', locals: { application: @application } %>
+
+<% if @application.adoption_description != nil %>
+  <h3>Why I'd make a good home: <%= @application.adoption_description %></h3>
+<% end %>
+
+<h3>Requested pet(s):</h3>
+<% @pets.each do |pet| %>
+  <section id="pet-<%= pet.id %>">
+    <table>
+      <tr>
+        <td>Name: </td>
+        <td><%= pet.name %></td>
+      </tr>
+      <tr>
+        <td>Age: </td>
+        <td><%= pet.age %></td>
+      </tr>
+      <tr>
+        <td>Breed: </td>
+        <td><%= pet.breed %></td>
+      </tr>
+      <tr>
+        <td>Shelter ID: </td>
+        <td><%= pet.shelter_id %></td>
+      </tr>
+    </table>
+    <% if @application.pet_status(pet) == nil %>
+      <br>
+      <%= button_to "Approve Pet", "/admin/applications/#{@application.id}?pet_id=#{pet.id}&status=approved", method: :patch, data: {turbo: false} %>
```

Comment:
Luckily, Turbo doesn't mess with buttons so you can leave the Turbo parameter out and it shouldn't break anything! Turbo needs to be disabled only for links and forms